pybaseball/teamid_lookup.py

```diff
@@ -57,56 +93,110 @@ def _generate_teams() -> pd.DataFrame:
     start_season = 1871
     end_season = most_recent_season()
 
+    lahman_columns = ['yearID', 'lgID', 'teamID', 'franchID', 'divID', 'name', 'teamIDBR', 'teamIDlahman45',
+                      'teamIDretro']
+
+    lahman_teams = lahman.teams()[lahman_columns]
+
     # Only getting AB to make payload small, and you have to specify at least one column
-    team_data = fangraphs.fg_team_batting_data(start_season, end_season, "ALL", stat_columns=['AB'])
-
-    # Join the lahman data
-    teams_franchises = lahman.teams().merge(lahman.teams_franchises(), how='left', on='franchID', suffixes=['', '.fr'])
-    teams_franchises = teams_franchises.merge(lahman.parks(), how='left', left_on='park', right_on='park.name',
-                                              suffixes=['', '.p'])
-
-    # Drop lahman data down to just what we need
-    teams_franchises = teams_franchises[
-        ['yearID', 'lgID', 'teamID', 'franchID', 'divID', 'name', 'park', 'teamIDBR', 'teamIDlahman45', 'teamIDretro',
-         'franchName', 'city', 'state']
-    ]
-
-    # Try to guess the name Fangraphs would use
-    teams_franchises['possibleName'] = teams_franchises.apply(lambda row: _estimate_name(row, 'name'), axis=1)
-    teams_franchises['possibleFranchName'] = teams_franchises.apply(lambda row: _estimate_name(row, 'franchName'),
-                                                                    axis=1)
-
-    # Join up the data by team name, and look for what is still without a match
-    outer_joined = teams_franchises.merge(team_data, how='outer', left_on=['yearID', 'possibleName'],
-                                          right_on=['Season', 'Team'])
-    unjoined_teams_franchises = outer_joined.query('Season.isnull()').drop(team_data.columns, axis=1)
-    unjoined_team_data = outer_joined.query('yearID.isnull()').drop(teams_franchises.columns, axis=1)
-
-    # Take all the unmatched data and try to join off franchise name, instead of team name
-    inner_joined = teams_franchises.merge(team_data, how='inner', left_on=['yearID', 'possibleName'],
-                                          right_on=['Season', 'Team'])
-    franch_inner_joined = unjoined_teams_franchises.merge(unjoined_team_data, how='inner',
-                                                          left_on=['yearID', 'possibleFranchName'],
-                                                          right_on=['Season', 'Team'])
+    fg_team_data = fangraphs.fg_team_batting_data(start_season, end_season, "ALL", stat_columns=['AB'])
+
+    fg_columns = list(fg_team_data.columns.values)
+
+    unjoined_team_data = fg_team_data.copy(deep=True)
```

Comment:
for consistency could we call `unjoined_team_data`  `unjoined_fangraphs_teams` (or `unjoined_fg_teams`)?